### PR TITLE
Implement New toolkit version csharp samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
**Sample List -** 

**1. msgext-link-unfurling/csharp
2. tab-channel-group-config-page-auth/csharp
3. msgext-link-unfurling-meeting/csharp
4. bot-tab-conversations/csharp
5. bot-tag-mention/csharp**

### CODE REVIEW

File: samples/msgext-link-unfurling-meeting/csharp/TeamsApp/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿{
+  "profiles": {
+    // Debug project within Teams
+    "Microsoft Teams (browser)": {
+      "commandName": "Project",
+      "launchUrl": "https://teams.microsoft.com/l/app/%TEAMSAPPID%?installAppPackage=true&webjoin=true&appTenantId=%TENANTID%&login_hint=%USERNAME%"
```
Comment (Alive-Fish) [inline samples/msgext-link-unfurling-meeting/csharp/TeamsApp/launchSettings.json:6]: The `launchUrl` seems wrong. Please use `${{TEAMS_APP_ID}}` replacing `%TEAMSAPPID%`, also, check other variables and `launchUrl`.

File: samples/msgext-link-unfurling/csharp/TeamsApp/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿{
+  "profiles": {
+    // Debug project within Teams
+    "Microsoft Teams (browser)": {
+      "commandName": "Project",
+      "launchUrl": "https://teams.microsoft.com/l/app/%TEAMSAPPID%?installAppPackage=true&webjoin=true&appTenantId=%TENANTID%&login_hint=%USERNAME%"
```
Comment (Alive-Fish) [inline samples/msgext-link-unfurling/csharp/TeamsApp/launchSettings.json:6]: The `launchUrl` seems wrong. Please use `${{TEAMS_APP_ID}}` replacing `%TEAMSAPPID%`, also, check other variables and `launchUrl`.

File: samples/tab-channel-group-config-page-auth/csharp/TeamsApp/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "profiles": {
+    // Debug project within Teams
+    "Microsoft Teams (browser)": {
+      "commandName": "Project",
+      "launchUrl": "https://teams.microsoft.com/l/app/%TEAMSAPPID%?installAppPackage=true&webjoin=true&appTenantId=%TENANTID%&login_hint=%USERNAME%"
```
Comment (Alive-Fish) [inline samples/tab-channel-group-config-page-auth/csharp/TeamsApp/launchSettings.json:6]: The `launchUrl` seems wrong. Please use `${{TEAMS_APP_ID}}` replacing `%TEAMSAPPID%`, also, check other variables and `launchUrl`.

**Review by Alive-Fish — APPROVED** [review]
Only find the following issues. Please check them.

File: samples/msgext-link-unfurling-meeting/csharp/TeamsApp/launchSettings.json (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/msgext-link-unfurling-meeting/csharp/TeamsApp/launchSettings.json:6]: Fixed

File: samples/msgext-link-unfurling/csharp/TeamsApp/launchSettings.json (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/msgext-link-unfurling/csharp/TeamsApp/launchSettings.json:6]: Fixed

File: samples/tab-channel-group-config-page-auth/csharp/TeamsApp/launchSettings.json (hunk 3 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-channel-group-config-page-auth/csharp/TeamsApp/launchSettings.json:6]: Fixed

**Review by Alive-Fish — APPROVED** [review]

**Review by Mohammed-MSFT — APPROVED** [review]

**Review by Jegadeesh-MSFT — APPROVED** [review]
Looks good !

**Pull request merged**